### DISCUSSION TITLE
VanPHP :)

### DISCUSSION THREAD

**Discussion by groovenectar** [vanjs-org/van#70] [Show and tell]
I've started on a proof-of-concept for generating a VanJS or HTML DOM tree using PHP:

https://github.com/groovenectar/VanPHP

I'm not sure yet what kind of direction it could take.

It produces a DOM tree like the convertor here: https://vanjs.org/convert

But written in PHP instead, and probably has a few caveats at the moment. One thought is that it could be used to generate SSR HTML and Van DOM with the same input. Or something else to do with a generated component pattern where properties are shared between the backend and frontend.

**Comment by Tao-VanJS** [comment]
Cool project! Thanks for your contribution to VanJS community!
